hack/generate/changelog/util/changelog_test.go

```diff
@@ -283,6 +283,7 @@ No changes for this release!
 	}
 
 	for _, tc := range testCases {
+		tc := tc
```

Comment:
What linter error is this fixing? I find it strange to make a copy of the variable in the loop. I've never had to do this.